spec/view_helper_spec.rb

```diff
@@ -51,6 +51,21 @@
         expect(meta).to have_tag('meta', with: { content: "facebook description", property: "og:description" })
       end
     end
+
+    it 'should work with app links parameters' do
+      subject.set_meta_tags('al' => {
+        'ios' => {
+          'url' => 'applinks://docs',
+          'app_store_id' => 12345,
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.